development/playbooks/deploy-dev/metadata.obsah.yaml

```diff
@@ -0,0 +1,17 @@
+---
+help: |
+  Deploy and manage Foreman development environment with git-based Foreman and containerized backend services.
+
+variables:
+  foreman_development_enabled_plugins:
+    help: List of plugins to enable from registry (comma-separated)
+    default: katello,foreman_remote_execution
```

Comment:
I wonder how this will play together with https://github.com/theforeman/foremanctl/pull/188 :)

Reply:
Good question, I can explore switching when we get there. I mostly just want to get people to using this setup.